beacon-chain/blockchain/metrics.go

```diff
@@ -317,8 +317,8 @@ func reportEpochMetrics(ctx context.Context, postState, headState state.BeaconSt
 	var b *precompute.Balance
 	var v []*precompute.Validator
 	var err error
-	switch headState.Version() {
-	case version.Phase0:
+	switch {
```

Comment:
I think this is better like this 
```go

switch v := headState.Version(); v {
   case v == version.Phase0:
   ...
   case v >= version.Altair:
   ...
}
```

Reply:
I like this but I got `Invalid case 'v == version.Phase0' in switch on 'v' (mismatched types 'bool' and 'int')`. Can't figure this out

Reply:
Pushed to use @potuz 's solution until we can figure out Prestin's suggestion

Reply:
ah yes, sorry, seems like you can't do what I suggested